index.js

```diff
@@ -0,0 +1,64 @@
+var request  = require('request');
+var schedule = require('node-schedule');
+var env      = require('node-env-file');
+
+// Load enviroment settings
+env(__dirname + '/.env');
+
+// Run once per day
+var cronSchedule = { hour: 00, minute: 00 }
+
+if(process.env.DEBUG) {
+  // Run every minute
+  cronSchedule = '*/1 * * * *';
+}
+
+// Schedule the job
+var j = schedule.scheduleJob(cronSchedule, function() { 
```

Comment:
Since updateAppGlanceSlice has no parameters, you should be able to use it as the callback method directly:

`var j = schedule.scheduleJob(cronSchedule, updateAppGlanceSlice);`